docs/reference/ccr/remote-recovery.asciidoc

```diff
@@ -0,0 +1,67 @@
+[role="xpack"]
+[testenv="platinum"]
+[[remote-recovery]]
+== Remote recovery
+
```

Comment:
I think you need a sentence or two of context at the beginning of this page. For example:

```suggestion
When you create a follower index, you cannot use it until it is fully initialized. 
```